.idea/codeStyles/Project.xml

```diff
@@ -23,6 +23,9 @@
     <Properties>
       <option name="KEEP_BLANK_LINES" value="true" />
     </Properties>
+    <ScalaCodeStyleSettings>
+      <option name="MULTILINE_STRING_CLOSING_QUOTES_ON_NEW_LINE" value="true" />
+    </ScalaCodeStyleSettings>
```

Comment:
Not sure why this started happening. This is for scala so I do not expect too much impact. If this ends up moving too much, I'll dig into it.